corehq/form_processor/management/commands/overwrite_blob_buckets.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.es import filters
+from corehq.apps.es.forms import FormES
+from corehq.blobs import get_blob_db
+from corehq.blobs.exceptions import NotFound
+from corehq.form_processor.backends.sql.dbaccessors import FormAccessorSQL
+
+logger = logging.getLogger('main')
+
+
+class Command(BaseCommand):
+    help = "Overwrites blob_bucket for attachments that aren't the default"
+
+    def handle(self, *args, **options):
+        possible_bad_forms = (
+            FormES()
+            .submitted(
+                gte=datetime.date(2016, 10, 14),
+                lte=datetime.datetime.now()
```

Comment:
this could have a hard end date when https://github.com/dimagi/commcare-hq/pull/13623 is deployed, correct?